roles/nginxplus/files/conf/http/templates/pudl_proxy_pass.conf

```diff
@@ -0,0 +1,307 @@
+# arks redirects
+    rewrite ^/objects/(.*)$ /objects/$1 redirect;
+    location /objects/ {
+        proxy_pass https://arks.princeton.edu/ark/88435/;
```

Comment:
Is this a proxy or a redirect? I was expecting to redirect them, but I don't know nginx config, so I'm not sure what this is doing.

Reply:
if this configuration lived on pudl dot princeton dot edu then a redirect would do. It lives on lib-adc dot princeton dot edu and proxy pass will work more reliably.